var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -12,9 +12,14 @@ class ParallelNetcdf(AutotoolsPackage):
     Unidata's NetCDF."""
 
     homepage = "https://trac.mcs.anl.gov/projects/parallel-netcdf"
-    url      = "http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.6.1.tar.gz"
-    list_url = "http://cucis.ece.northwestern.edu/projects/PnetCDF/download.html"
+    git      = "https://github.com/Parallel-NetCDF/PnetCDF"
+    url      = "https://parallel-netcdf.github.io/Release/parallel-netcdf-1.11.0.tar.gz"
```

Comment:
I would change the `url` to use `pnetcdf` instead of `parallel-netcdf`. That way, `spack versions` and `spack checksum` can be used to add future versions. We may also want to consider renaming the package to `pnetcdf`. To avoid having to manually specify the URL for previous releases, add a `url_for_version` function. Let me know if you need help writing this.